examples/MultiXcodeProject Example/Podfile

```diff
@@ -22,3 +23,5 @@ target 'MultiXcodeProjFrameworks' do
   pod 'Coffee', path: 'Coffee/Coffee.podspec'
   pod 'CoffeeMachine', path: 'CoffeeMachine/CoffeeMachine.podspec'
 end
+
+ENV['COCOAPODS_DISABLE_STATS'] = 'true'
```

Comment:
is this added to all examples?

Reply:
Nope. Can do separately. Rather keep this diff smaller